src/renderer/modules/webpack/common/modal.ts

```diff
@@ -0,0 +1,83 @@
+import { ModuleExports, ObjectExports } from "../../../../types/discord";
```

Comment:
What's the difference between this and the modal component? Shouldn't those just be combined?

Reply:
This has the open, close, etc. These make the modals appear on the screen. The components is what is shown. They're different